app/controllers/movies_controller.rb

```diff
@@ -34,6 +33,58 @@ def show
       vote_average: 8.2,
       title: 'F9',
       runtime: 138,
+      genres: [
+        {
+          id: 28,
+          name: 'Action'
+        },
+        {
+          id: 878,
+          name: 'Science Fiction'
+        }
+      ],
+      cast: [
+        { id: 73457,
+          name: 'Chris Pratt',
+          profile_path: '/gXKyT1YU5RWWPaE1je3ht58eUZr.jpg',
+          character: 'Dan Forester' },
+        { id: 73457,
+          name: 'Chris Pratt',
+          profile_path: '/gXKyT1YU5RWWPaE1je3ht58eUZr.jpg',
```

Comment:
What is this bit? Is it just a placeholder to make sure this will work evenutally?